src/audio/codec_adapter/codec_adapter.c

```diff
@@ -225,22 +225,21 @@ int codec_adapter_params(struct comp_dev *dev,
 }
 
 static void
-codec_adapter_copy_from_source_to_lib(const struct audio_stream *source,
-				      const struct module_processing_data *mpd,
-				      size_t bytes)
+ca_copy_from_source_to_module(const struct audio_stream *source, void *buff, uint32_t buff_size,
+			      size_t bytes)
 {
```

Comment:
I'll go with signature modification in a first patch then rename in a second one. It is easy to follow. 

Also, while doing the second patch (rename function) maybe it is worth to add some doc style comments to this function because it is not clear what buff_size and bytes are.